### PR TITLE
Fix DB update issues when updating from old Limesurvey versions

### PR DESCRIPTION
**Pull request by mbischof**
Fixed issue # : 15759
New feature # :
Changed feature # :
Dev:

### CODE REVIEW

**Comment by maziminke** [comment]
There is also a bug report at https://bugs.limesurvey.org/view.php?id=15759.

**Comment by Shnoulle** [comment]
Question : maybe the existing table must be renamed (or deleted) and created again ?

**Comment by Shnoulle** [comment]
Or better : just remove this createBoxes …

Created again : https://github.com/LimeSurvey/LimeSurvey/blob/50ab52cf35fb5e5f146782592211ae4aacdd783b/application/helpers/update/updatedb_helper.php#L1946-L1961

at dbversion 338

File: application/helpers/update/updatedb_helper.php [hunk 1]
```diff
@@ -1339,6 +1339,7 @@ function db_upgrade_all($iOldDBVersion, $bSilent = false)
         if ($iOldDBVersion < 256) {
             $oTransaction = $oDB->beginTransaction();
             upgradeTokenTables256();
+            $oDB->createCommand()->update('{{participants}}', array('email' => ''), 'email IS NULL');
```
Comment (Shnoulle) [inline application/helpers/update/updatedb_helper.php]: Think is not related to this issue …


I do same … staring from a fork to do another fork : https://github.com/LimeSurvey/LimeSurvey/pull/1370#issuecomment-578443506 :)

Comment (olleharstedt) [reply application/helpers/update/updatedb_helper.php]: If this is not related to the issue, please remove from this PR and add it to another one.

**Comment by maziminke** [comment]
Both commits deal with DB update issues when updating from old versions. We will simply rename the pull request to "Fix DB update issues when updating from old Limesurvey versions".

**Comment by maziminke** [comment]
Please review and merge, thanks.

**Comment by olleharstedt** [comment]
I already requested a change.

**Comment by maziminke** [comment]
@olleharstedt Seriously, this is simply fixing issues during updating from old versions. There is no complexity. Why that additional work of separating it into different pull requests? 
There is no benefit and dealing with such really minor fixes that way will lead to less fixes by the community or Limesurvey partners. It is not worth the effort.

**Comment by olleharstedt** [comment]
Ah, I was mostly following Denis' advice that it was unrelated. A commit should be a unit of work that does only one thing. This makes the git history more clear. But I will merge this. The commit is so small anyway.

**Comment by olleharstedt** [comment]
Thank you for the contribution!

**Comment by maziminke** [comment]
Thanks, Olle!

**Pull request merged**